File: 3-Chapter-4.1/Revision_Material/three.cs
using System;
namespace CSharpWork
{
    public class Three
    {
        public Three()
        {
            int one = IntPrompt();
            int two = IntPrompt();
            int three = IntPrompt();

            if (one > two && one > three)
            {
                Output(one + " is biggest");
            }
            else if (two > one && two > three)
            {
                Output(two + " is biggest");
            }
            else if (three > one && three > two)
            {
                Output(three + " is biggest");
            }
            else
            {
                Output("All the numbers are the same.");
            }
        }

        static void Output(string value)
        {
            Console.WriteLine(value);
        }

        static int IntPrompt()
        {
            Console.WriteLine("Enter an integer:");
            return int.Parse(Console.ReadLine());
        }
    }
}

File: 3-Chapter-4.1/Revision_Material/three.py
one = int(input("Enter an integer:"))
two = int(input("Enter an integer:"))
three = int(input("Enter an integer:"))


def sum(first, second, third):
    return first + second + third


def output(value):
    print(value)


if one > two and one > three:
    output(f"{one} is biggest")
elif two > one and two > three:
    output(f"{two} is bigger")
elif three > one and three > two:
    output(f"{three} is bigger")
else:
    output("All the numbers are the same.")
